templates/webapi/test/tr_job_result_details.html.ep

```diff
@@ -18,3 +18,11 @@
         %= comment_icon($jobid, $res->{comments});
     </span>
 % }
+% my $text = show_job_deps($res->{deps});
+% if ($text) {
+    <a href="/tests/<%= $jobid %>#dependencies" title="<%= $text %>"><i class="fa fa-code-branch"></i></a>
+% }
+
```

Comment:
the screenshot shows a bit spacing between the fa-code-branch icon and fa-clone, maybe due to this blank line?

Anyway, as in my previous comment it might be easier if we focus on the branch icon only first